R/utils.R

```diff
@@ -33,6 +33,25 @@ is_syntactic <- function(x) {
   x == make.names(x)
 }
 
+is_hex_color <- function(x) {
```

Comment:
Maybe move this and the to-be-implemented color helpers to a new file  `colors.R` ?